src/java-interop/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+set(CMAKE_OSX_ARCHITECTURES x86_64 arm64)
+
+project(
+    java-interop
+    DESCRIPTION "Java.Interop native support"
+    HOMEPAGE_URL "https://github.com/xamarin/java.interop/"
+    LANGUAGES CXX C
+)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+
```

Comment:
You might want to set C standard to something modern (e.g. 11)

Reply:
Not sure this is needed?  All source code is now C++, and there's a `set(CMAKE_CXX_STANDARD 17)` statement.

Reply:
Wasn't there some generated .c file?

Reply:
ooh, right…